src/sqlfluff/core/rules/std/L043.py

```diff
@@ -0,0 +1,151 @@
+"""Implementation of Rule L043."""
+
+from sqlfluff.core.rules.base import BaseCrawler, LintFix, LintResult
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
+
+
+@document_fix_compatible
+class Rule_L043(BaseCrawler):
+    """Use an explicit condition rather than case when statement.
```

Comment:
Can you think of a more precise way to explain the recommended practice (mentioning that it's not just "don't use case", but also about what the "case" statement is doing.

Reply:
Ah yeah that makes sense. I rewrote this a little. Hopefully it's clearer but not too declarative either.